apps/allocations/app/components/App/App.js

```diff
@@ -42,14 +71,21 @@ const App = () => {
     closePanel()
   }
 
+  const onSubmitDeactivate = (id) => {
+    console.log(`deactivating budget # ${id}...`)
+    //api.deactivateBudget(id)
+    closeModal()
+  }
+
   const onExecutePayout = (accountId, payoutId) => {
     api.runPayout(accountId, payoutId).toPromise()
   }
 
   const onNewBudget = () => {
+    const fundsLimit = '300000' // remove this!
     setPanel({
       content: NewBudget,
-      data: { heading: 'New budget', onCreateBudget }
+      data: { heading: 'New budget', onCreateBudget, fundsLimit }
```

Comment:
If we remove fundsLimit, where will this value come from?

Reply:
I think it's the total amount the DAO has in its vault, minus the sum of the amount of every Budget. Maybe it should be an array of objects, each one representing the funds limit of every token.